server/fleet/app.go

```diff
@@ -1037,6 +1037,9 @@ type ApplySpecOptions struct {
 	DryRun bool
 	// TeamForPolicies is the name of the team to set in policy specs.
 	TeamForPolicies string
+	// ExpandEnvConfigProfiles enables expansion of environment variables in
+	// configuration profiles.
+	ExpandEnvConfigProfiles bool
```

Comment:
Not sure this should be in `ApplySpecOptions`. These options are used by `service.go` methods, but `ExpandEnvConfigProfiles` is only relevant for fleetctl.

Maybe create a `ApplyFleetctlSpecOptions` supertype? I recently created `ApplyTeamSpecOptions` using similar logic.

Reply:
Good point. Added a `ApplyClientSpecOptions` type.